### PR TITLE
mate-panel-applet-factory: Boilerplate

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

**Review by lukefromdc — COMMENTED** [review]
This works, but please fix the misaligned indents in the changes to mate-panel-applet-factory.c

**Comment by rbuj** [comment]
done

**Comment by lukefromdc** [comment]
We still get some misalignment, I cannot copy and paste it here because the browser moves it more. lines 34,36, 40 in mate-panel-applet-factory.c 
I do NOT know if the * is intentionally meant to b offset to the left one space however

**Comment by rbuj** [comment]
I don't see it misaligned.
Screenshot at 2020-11-21 12-52-38
Did you use a monospaced font for viewing the file?

**Comment by lukefromdc** [comment]
I will step back from further review of indents on this PR. I do not have a sufficient understanding of fonts in the browser to answer your question. 


I do not deliberately use a monospace font anywhere outside the terminal but I know little of browser and website font behavior beyond the need to block websites from downloading fonts for security reasobs.

**Comment by raveit65** [comment]
> In your screenshot from Pluma (not visible on my phone browser due to 3ed party content block) I see the * characters positioned left of everything else, and I do NOT know if that is correct or not, so again I need someone else to judge this

This is fine, same like in other files/applications/repos of us.

File: libmate-panel-applet/mate-panel-applet-factory.c [hunk 1]
```diff
@@ -285,35 +278,41 @@ static const GDBusInterfaceVTable interface_vtable = {
...
-					   &interface_vtable,
-					   factory, NULL,
-					   &error);
+	factory->registration_id
+		= g_dbus_connection_register_object (connection,
+                                                     object_path,
+                                                     introspection_data->interfaces[0],
+                                                     &interface_vtable,
+                                                     factory, NULL,
+                                                     &error);
```
Comment (raveit65) [inline libmate-panel-applet/mate-panel-applet-factory.c:302]: Doesn't align with `c of connection` line 297. Same when i open this commit in pluma (tabsize=8)

I think you will run into pain with mixed-usage of tabs and spaces all over this file.

**Comment by raveit65** [comment]
Seems like most of the commit is code formatting.
Honestly, i don't see what the real change is.
Maybe, it's better to split the commit?

**Comment by rbuj** [comment]
@raveit65 done - split commit

**Comment by raveit65** [comment]
Thanks,
so this mainly affects applets when build out-of-process, right?
Than i need to change how i build the panel for testing.

**Review by raveit65 — APPROVED** [review]
I saw no issues after several days of testing.

**Review by lukefromdc — APPROVED** [review]
Good to go on this end, dropping indent concerns due to not enough knowledge about how others want that * aligned. merging

**Pull request merged**